unitary/examples/quantum_rpg/qaracter.py

```diff
@@ -93,44 +93,49 @@ def add_quantum_effect(self, effect, quantum_name: Union[int, str]):
         hp_obj = self.get_hp(quantum_name)
         effect(hp_obj)
 
+    @property
+    def damage(self) -> int:
+        """Returns the number of qubits measured to be zero."""
+        return sum(status == 0 for status in self.health_status.values())
+
+    @property
+    def virtue(self) -> int:
+        """Returns the number of qubits measured to be one."""
+        return sum(status == 1 for status in self.health_status.values())
+
     def is_down(self) -> bool:
         """Returns True if over half of the HP have been measured as `HURT`."""
-        damage = len(
-            [q for q in self.health_status if self.health_status[q] == 0])
-        return damage > (self.level / 2)
+        return self.damage > (self.level / 2)
 
     def is_escaped(self) -> bool:
         """Returns True if all HP are measured and at least half are `HEALTHY`."""
```

Comment:
Neither is_escaped or is_down is True right now for `damage == virtue == level/2`. Maybe add a test for an even number of HP.

Reply:
Fixed and added case for even qubits.  Thanks for catching it!